src/ansi.rs

```diff
@@ -1551,6 +1554,28 @@ where
 
                 handler.clear_line(mode);
             },
+            ('k', [b' ']) => {
+                // SCP control
+                //
+                // Changes from ECMA-48 as proposed by the BiDi draft proposal:
+                //  * Recognize value 0 for Ps1
+                //  * Allow passing no params or Ps1 alone
+                //  * Assume default 0 value for both Ps1 and Ps2
+                //
+                //  https://terminal-wg.pages.freedesktop.org/bidi/recommendation/escape-sequences.html
+                let ps1 = params_iter.next().unwrap_or(&[0]);
+                let ps2 = params_iter.next().unwrap_or(&[0]);
+                if params_iter.next().is_some() {
+                    unhandled!();
```

Comment:
Not exactly wrong, but looking at other escapes we have been mostly ignoring additional parameters.

Might be nice in case someone comes along and adds a new parameter to do whatever they seem is necessary, so we still support it without having to explicitly handle that parameter.